test/lib/mayaUsd/render/mayaToHydra/testMtohVisibility.py

```diff
@@ -1,7 +1,3 @@
-import os
```

Comment:
`os` became unused after my changes below, but `sys` and `unittest` were never used, so I just removed all three.